docs/contributor/04-10-technical-design.md

```diff
@@ -33,7 +33,7 @@ Therefore, a user who needs such external monitoring must take care of this part
 Global mTLS PeerAuthentication is tightly coupled with the Istio installation. Therefore, it should be reconciled by the operator.
 
 ##### Kyma Gateway
-Kyma Gateway is moved to the API Gateway, as it is a default gateway we want to provide, and its responsibilities are more closely connected
+Kyma Gateway is moved to the API Gateway, as it is a default gateway we provide, and its responsibilities are more closely connected
```

Comment:
[Our guidelines are against "we"](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md)